issue compiling roapi-http v0.1.13  could not compile `actix-cors`

Hello,
I try to compile roapi-http from the [source zip](https://github.com/roapi/roapi/archive/refs/tags/roapi-http-v0.1.3.zip)

I have an error when I compile 

`cargo install --verbose --path ./roapi-http --bin roapi-http`
building
...
...

```
error[E0053]: method `error_response` has an incompatible type for trait                                                              │
  --> /usr/local/cargo/git/checkouts/actix-extras-3214827614a42f65/ab3bdb6/actix-cors/src/error.rs:53:5                               │
   |                                                                                                                                  │
53 |     fn error_response(&self) -> HttpResponse {                                                                                   │
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected struct `BaseHttpResponse`, found struct `actix_web::HttpResponse`          │
   |                                                                                                                                  │
   = note: expected fn pointer `fn(&CorsError) -> BaseHttpResponse<actix_web::dev::Body>`                                             │
              found fn pointer `fn(&CorsError) -> actix_web::HttpResponse`                                                            │
                                                                                                                                      │
error: aborting due to previous error                                                                                                 │
                                                                                                                                      │
For more information about this error, try `rustc --explain E0053`.                                                                   │
error: could not compile `actix-cors`                                                                                                 │
```

rust version 1.51 because 1.50 gives me another set of error with flatbuffers regarding const generics:

error[E0658]: const generics are unstable
   |
28 | pub struct Array<'a, T: 'a, const N: usize>(&'a [u8], PhantomData<T>);
   |                                   ^
   |
   = note: see issue #74878 <https://github.com/rust-lang/rust/issues/74878> for more information

--- a/roapi-http/Cargo.toml
+++ b/roapi-http/Cargo.toml
@@ -17,9 +17,9 @@ columnq = { path = "../columnq" }
 arrow  = "4.3.0"
 
 # all actix dependencies are patched to use git source until actix-web 4.x lands
-actix-web = "4.0.0-beta"
-actix-http = "3.0.0-beta"
-actix-service = "2.0.0-beta"
+actix-web = "4.0.0-beta.4"
+actix-http = "3.0.0-beta.4"
+actix-service = "2.0.0-beta.3"
 # see https://github.com/actix/actix-extras/pull/144 for cors tokio 1 upgrade
 actix-cors = "*"
 

